core/banking/transfers_common.go

```diff
@@ -180,8 +184,13 @@ func (e *Engine) makeFeeTransferForTransferFunds(
 	from, asset string,
 	fromAccountType types.AccountType,
 ) *types.Transfer {
+	// no fee for Vested account
+	feeAmount := num.UintZero()
+
 	// first we calculate the fee
-	feeAmount, _ := num.UintFromDecimal(amount.ToDecimal().Mul(e.transferFeeFactor))
+	if fromAccountType != types.AccountTypeVestedRewards {
```

Comment:
I think the requirement is that this applies only if you're transferring between your own account, so if you transfer from vested to some other general account (not yours) there should still be fee (according to the ticket at least).

Reply:
Fair point.